api/tests/unit/features/test_unit_features_views.py

```diff
@@ -55,7 +55,7 @@ def test_list_feature_states_from_simple_view_set(
     )
 
     # When
-    with django_assert_num_queries(7):
+    with django_assert_num_queries(8):
```

Comment:
It's because of the query here: `https://github.com/Flagsmith/flagsmith/pull/2232/files#diff-7904b170017aa6b6a88dae1dc0905d9308c5b7b9475020895a44427433d7a60cR77` earlier it was using this function here: https://github.com/Flagsmith/flagsmith/pull/2232/files#diff-c2ac9247f7c75c1415fd7b42bc9ea9b8e62b83abfedc455585a807fe14f5c09bL303

Reply:
Hmm... but that logic already existed here, right? 

https://github.com/Flagsmith/flagsmith/pull/2232/files#diff-c2ac9247f7c75c1415fd7b42bc9ea9b8e62b83abfedc455585a807fe14f5c09bL340-L341

Reply:
Correct, but it was not using that function - it was using this function here: https://github.com/Flagsmith/flagsmith/pull/2232/files#diff-c2ac9247f7c75c1415fd7b42bc9ea9b8e62b83abfedc455585a807fe14f5c09bL297